components/SignIn.js

```diff
@@ -239,8 +239,8 @@ export default class SignIn extends React.Component {
                     type="password"
                     width={1}
                     value={password}
-                    autoFocus={this.props.passwordRequired ? true : false}
-                    required={this.props.passwordRequired ? true : false}
+                    autoFocus={this.props.passwordRequired}
+                    required={this.props.passwordRequired}
```

Comment:
This will introduce a change here, as we'll pass `false` instead of `undefined` in some cases. Any pointer on how it's related to the root issue?

Reply:
Sorry, I thought I was simplifying here when I saw this but didn't think about this undefined case. I've reverted this. 👍🏽